ionoscloud/resource_datacenter.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
```

Comment:
@cristiGuranIonos I also noticed now that we don't have any tests for the new datacenter attributes. Sorry, I missed this in the first round of review.

Reply:
removed them as we're now using a resource to attach NSGs to datacenters